exotic/exotic.py

```diff
@@ -3215,7 +3232,9 @@ def main():
                         'Am1': {'value': str(round_to_2(myfit.parameters['a1'], myfit.errors['a1'])),
                                 'uncertainty': str(round_to_2(myfit.errors['a1']))},
                         'Am2': {'value': str(round_to_2(myfit.parameters['a2'], myfit.errors['a2'])),
-                                'uncertainty': str(round_to_2(myfit.errors['a2']))}}
+                                'uncertainty': str(round_to_2(myfit.errors['a2']))},
+                        'Duration':{'value':np.round(np.mean(durs),5),
```

Comment:
please change to:
`"Duration":f"{round_to_2(np.mean(durs),np.std(durs))} +/- {round_to_2(np.std(durs))}"`